independent-publisher-connectors/Infura Ethereum/README.md

```diff
@@ -0,0 +1,59 @@
+# Infura Ethereum Connector
+
+Infura provides services to access the Ethereum Blockchain without your own Ethereum Node. This connector allows you to interact with the Ethereum Blockchain JSON-RPC API through Infura. Common use cases include checking your Ethereum account's (Wallet) balance, current gas fees, or information on the current block.
+
```

Comment:
@sebastianzolg Please refer the [template provided for readme.md file](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md) and add required sections.

Reply:
Added Publisher Name, Known Issues, and Deployment Instructions.